applications/plugins/LMConstraint/LMConstraintConfig.cmake.in

```diff
@@ -0,0 +1,23 @@
+# CMake package configuration file for the plugin @PROJECT_NAME@
+
+@PACKAGE_GUARD@
+@PACKAGE_INIT@
+
+set(LMCONSTRAINT_HAVE_SOFADISTANCEGRID @LMCONSTRAINT_HAVE_SOFADISTANCEGRID@)
+
+find_package(SofaFramework REQUIRED)
+find_package(SofaBase REQUIRED)
+find_package(SofaCommon REQUIRED) 
+find_package(SofaConstraint REQUIRED) 
+
+
+if(LMCONSTRAINT_HAVE_SOFADISTANCEGRID)
+    find_package(SofaDistanceGrid QUIET) 
```

Comment:
if it has DistanceGrid, it MUST find it. Then find_package(SofaDistanceGrid **REQUIRED**)